dist/s_funcs.list

```diff
@@ -3,6 +3,7 @@ WT_CRC32_ENTRY
 WT_CURDUMP_PASS
 __bit_ffs
 __bit_nclr
+__layered_drain_ingest_tables
```

Comment:
@quchenhao  - this change overlaps with SLS-808. I didn't want to mess with your changes, so I've added an exception for this function. Could you clean this up if the change here gets merged before SLS-808?

Reply:
sure, understand